package.json

```diff
@@ -1,13 +1,15 @@
 {
   "name": "water.css",
-  "version": "2.0.0",
+  "version": "1.4.0",
   "description": "A just-add-css collection of styles to make simple websites just a little nicer.",
```

Comment:
As with the repository's description, this needs to be updated to the newer ("drop-in") tag line.

Reply:
Fixed! I also updated the description on GitHub.